tokio-postgres/tests/test/main.rs

```diff
@@ -347,7 +347,250 @@ async fn custom_range() {
+            _ => panic!("unexpected message"),
+        }
+        match &messages[3] {
+            SimpleQueryMessage::CommandComplete(CommandCompleteContents { fields: f, .. }) => {
+                if f.is_some() {
+                    panic!("Unexpected data found");
+                }
+            }
+            _ => panic!("unexpected message"),
+        }
+        match &messages[4] {
+            SimpleQueryMessage::CommandComplete(CommandCompleteContents { fields: f, .. }) => {
+                if let Some(field_vec) = &f {
+                    assert_eq!((&**field_vec).len(), 1);
+                    assert_eq!("k", (&**field_vec)[0].name());
+                } else {
+                    panic!("No data found");
+                }
+            }
+            _ => panic!("unexpected message"),
+        }
+        match &messages[5] {
+            SimpleQueryMessage::CommandComplete(CommandCompleteContents { fields: f, .. }) => {
+                if f.is_some() {
+                    panic!("Unexpected data found");
+                }
+            }
+            _ => panic!("unexpected message"),
+        }
+        match &messages[6] {
+            SimpleQueryMessage::CommandComplete(CommandCompleteContents { fields: f, .. }) => {
+                if f.is_some() {
+                    panic!("Unexpected data found");
+                }
+            }
+            _ => panic!("unexpected message"),
+        }
+        match &messages[7] {
+            SimpleQueryMessage::CommandComplete(CommandCompleteContents { fields: f, .. }) => {
+                if f.is_some() {
+                    panic!("Unexpected data found");
+                }
+            }
+            _ => panic!("unexpected message"),
+        }
+        match &messages[8] {
+            SimpleQueryMessage::CommandComplete(CommandCompleteContents { fields: f, .. }) => {
+                if f.is_some() {
+                    panic!("Unexpected data found");
+                }
+            }
+            _ => panic!("unexpected message"),
+        }
+
+        assert_eq!(messages.len(), 9);
+    }
+    match client
+        .simple_query("CREATE TABLE sbtest2 (id INTEGER, k INTEGER);")
+        .await
+    {
+        Ok(_) => {
+            println!("Query executed successfully");
+            // Your code here when the query succeeds
+        }
+        Err(e) => {
+            eprintln!("Error executing query: {}", e);
+            // Your code here when the query fails
+        }
+    }
 
+    let messages = client
+        .simple_query(
+            "INSERT INTO sbtest1  VALUES (1, 2);
+            INSERT INTO sbtest2  VALUES (1, 2);
+            SELECT * FROM sbtest1 ORDER BY id;
+            SELECT k FROM sbtest1 WHERE id = 999;
+            BEGIN;
+            UPDATE sbtest1 SET k=id;
+            UPDATE sbtest2 SET k=id;
+            END;",
+        )
+        .await
+        .unwrap();
+
+    println!("{:?}", messages);
+    let _m = format!("{:?}", messages);
```

Comment:
Is this used?

Reply:
I had some merge issues and it seems like some of my changes did not make it into the review.